packages/gatsby/src/utils/webpack-utils.ts

```diff
@@ -350,18 +341,6 @@ export const createWebpackUtils = (
       }
     },
 
-    url: (options = {}) => {
-      return {
-        loader: require.resolve(`url-loader`),
-        options: {
-          limit: 10000,
-          name: `${assetRelativeRoot}[name]-[hash].[ext]`,
-          fallback: require.resolve(`file-loader`),
-          ...options,
-        },
-      }
-    },
-
```

Comment:
this removes `loader.url` which I think should be considered public API. Any chance we can keep it (if 3rd party plugins use it) for backward compat even if we don't use it ourselves?

Reply:
Sure! Didn't think people can access that 😬